README.md

```diff
@@ -1,8 +1,8 @@
-# Curassow
+# Currasow
```

Comment:
Hmm, Curassow is right, the repo name is wrong. Please drop these changes and I'll deal with the name problem.